mm/routines/product.py

```diff
@@ -17,11 +17,18 @@ def update_enum_product_attribute(self, sku, att_name, att_value):
         return record
 
     def get_configurable_associated_products(self, store_id=None, is_active=None, is_visible=None):
+        legal_inputs = set([None, True, False])
+        message = "Not a valid input value for '{0}'. Use: {1}"
+
+        assert type(store_id) is int or store_id is None, message.format('store_id', 'None or int')
+        assert is_active in legal_inputs, message.format('is_active', 'None, True or False')
```

Comment:
I get that, In Python `bool `is a subclass of an `int`. There should be no difference between 1 and True or 0 and False. 
I could have done `is_active in [1,0,None]`